### ISSUE TITLE
Can not use the plugin with a new version of the Strapi CMS (4.1.5)

### ISSUE THREAD

**Issue by AlexDede** [boazpoolman/strapi-plugin-config-sync#46]
### Describe the bug

This plugin is not compatible with the new version (4.1.5) of Strapi CMS. 

The error during execute `yarn build` command

```bash

$ strapi build
Building your admin UI with development configuration ...

● Webpack █████████████████████████ building (24%) 0/1 entries 6200/6260 dependencies 1363/1754 modules 272 active 
 babel-loader › node_modules/@strapi/plugin-content-type-builder/admin/src/components/AllowedTypesSelect/index.js

ModuleNotFoundError: Module not found: Error: Can't resolve '@strapi/design-system/Text' in '/Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/strapi-plugin-config-sync/admin/src/components/ConfigList'
    at /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/webpack/lib/Compilation.js:2011:28
    at /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/webpack/lib/NormalModuleFactory.js:795:13
    at eval (eval at create (/Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:10:1)
    at /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/webpack/lib/NormalModuleFactory.js:275:22
    at eval (eval at create (/Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
    at /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/webpack/lib/NormalModuleFactory.js:431:22
    at /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/webpack/lib/NormalModuleFactory.js:124:11
    at /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/webpack/lib/NormalModuleFactory.js:667:25
    at /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/webpack/lib/NormalModuleFactory.js:852:8
    at /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/webpack/lib/NormalModuleFactory.js:972:5
    at /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/neo-async/async.js:6883:13
    at /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/webpack/lib/NormalModuleFactory.js:955:45
    at finishWithoutResolve (/Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/enhanced-resolve/lib/Resolver.js:307:11)
    at /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/enhanced-resolve/lib/Resolver.js:381:15
    at /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/enhanced-resolve/lib/Resolver.js:430:5
    at eval (eval at create (/Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
resolve '@strapi/design-system/Text' in '/Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/strapi-plugin-config-sync/admin/src/components/ConfigList'
  Parsed request is a module
  using description file: /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/strapi-plugin-config-sync/package.json (relative path: ./admin/src/components/ConfigList)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/strapi-plugin-config-sync/admin/src/components/ConfigList/node_modules doesn't exist or is not a directory
      /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/strapi-plugin-config-sync/admin/src/components/node_modules doesn't exist or is not a directory
      /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/strapi-plugin-config-sync/admin/src/node_modules doesn't exist or is not a directory
      /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/strapi-plugin-config-sync/admin/node_modules doesn't exist or is not a directory
      looking for modules in /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/strapi-plugin-config-sync/node_modules
        /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/strapi-plugin-config-sync/node_modules/@strapi/design-system doesn't exist
      /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/node_modules doesn't exist or is not a directory
      looking for modules in /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules
        existing directory /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/@strapi/design-system
          using description file: /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/@strapi/design-system/package.json (relative path: .)
            using description file: /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/@strapi/design-system/package.json (relative path: ./Text)
              no extension
                Field 'browser' doesn't contain a valid alias configuration
                /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/@strapi/design-system/Text doesn't exist
              .js
                Field 'browser' doesn't contain a valid alias configuration
                /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/@strapi/design-system/Text.js doesn't exist
              .jsx
                Field 'browser' doesn't contain a valid alias configuration
                /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/@strapi/design-system/Text.jsx doesn't exist
              .react.js
                Field 'browser' doesn't contain a valid alias configuration
                /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/@strapi/design-system/Text.react.js doesn't exist
              as directory
                /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/@strapi/design-system/Text doesn't exist
      /Users/alex.dede/projects/cdp/node_modules doesn't exist or is not a directory
      /Users/alex.dede/projects/node_modules doesn't exist or is not a directory
      /Users/alex.dede/node_modules doesn't exist or is not a directory
      /Users/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
      looking for modules in /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/@strapi/admin/node_modules
        /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/@strapi/admin/node_modules/@strapi/design-system doesn't exist
ModuleNotFoundError: Module not found: Error: Can't resolve '@strapi/design-system/Text' in '/Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/strapi-plugin-config-sync/admin/src/components/ConfigList'
    at /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/webpack/lib/Compilation.js:2011:28
    at /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/webpack/lib/NormalModuleFactory.js:795:13
    at eval (eval at create (/Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:10:1)
    at /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/webpack/lib/NormalModuleFactory.js:275:22
    at eval (eval at create (/Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
    at /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/webpack/lib/NormalModuleFactory.js:431:22
    at /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/webpack/lib/NormalModuleFactory.js:124:11
    at /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/webpack/lib/NormalModuleFactory.js:667:25
    at /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/webpack/lib/NormalModuleFactory.js:852:8
    at /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/webpack/lib/NormalModuleFactory.js:972:5
    at /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/neo-async/async.js:6883:13
    at /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/webpack/lib/NormalModuleFactory.js:955:45
    at finishWithoutResolve (/Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/enhanced-resolve/lib/Resolver.js:307:11)
    at /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/enhanced-resolve/lib/Resolver.js:381:15
    at /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/enhanced-resolve/lib/Resolver.js:430:5
    at eval (eval at create (/Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
resolve '@strapi/design-system/Text' in '/Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/strapi-plugin-config-sync/admin/src/components/ConfigList'
  Parsed request is a module
  using description file: /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/strapi-plugin-config-sync/package.json (relative path: ./admin/src/components/ConfigList)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/strapi-plugin-config-sync/admin/src/components/ConfigList/node_modules doesn't exist or is not a directory
      /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/strapi-plugin-config-sync/admin/src/components/node_modules doesn't exist or is not a directory
      /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/strapi-plugin-config-sync/admin/src/node_modules doesn't exist or is not a directory
      /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/strapi-plugin-config-sync/admin/node_modules doesn't exist or is not a directory
      looking for modules in /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/strapi-plugin-config-sync/node_modules
        /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/strapi-plugin-config-sync/node_modules/@strapi/design-system doesn't exist
      /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/node_modules doesn't exist or is not a directory
      looking for modules in /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules
        existing directory /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/@strapi/design-system
          using description file: /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/@strapi/design-system/package.json (relative path: .)
            using description file: /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/@strapi/design-system/package.json (relative path: ./Text)
              no extension
                Field 'browser' doesn't contain a valid alias configuration
                /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/@strapi/design-system/Text doesn't exist
              .js
                Field 'browser' doesn't contain a valid alias configuration
                /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/@strapi/design-system/Text.js doesn't exist
              .jsx
                Field 'browser' doesn't contain a valid alias configuration
                /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/@strapi/design-system/Text.jsx doesn't exist
              .react.js
                Field 'browser' doesn't contain a valid alias configuration
                /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/@strapi/design-system/Text.react.js doesn't exist
              as directory
                /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/@strapi/design-system/Text doesn't exist
      /Users/alex.dede/projects/cdp/node_modules doesn't exist or is not a directory
      /Users/alex.dede/projects/node_modules doesn't exist or is not a directory
      /Users/alex.dede/node_modules doesn't exist or is not a directory
      /Users/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
      looking for modules in /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/@strapi/admin/node_modules
        /Users/alex.dede/projects/cdp/cdp-scpc-cms/node_modules/@strapi/admin/node_modules/@strapi/design-system doesn't exist
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

### Steps to reproduce the behavior

1. Create a new Strapi (4.1.5) project
2. Add `strapi-plugin-config-sync` plugin to the created project
3. Execute `yarn build` command
4. See error in the console

### Expected behavior

The project should be built without any errors

### System

- Node.js version: 16.14.0
- NPM version:  8.3.1
- Strapi version: 4.1.5
- Plugin version: 1.0.0-beta.8
- Database: MariaDB
- Operating system: Mac OS

**Comment by dzcpy** [comment]
Same issue. Please fix it, thanks!

**Comment by boazpoolman** [comment]
Thanks for bringing up the issue.
I'm on it.

**Comment by portu89** [comment]
im having this same issue, and im using version 4.1.3

**Comment by boazpoolman** [comment]
Fixed in `v1.0.0` 😄
